### PR TITLE
HTML examples

### LINKED ISSUE #737: [Edits] HTML: Output of examples
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

Many entries in the HTML section do not have the output of the example.

## List of entries with no output of the example

[`<video>`](https://www.codecademy.com/resources/docs/html/tags/video)
[`<strong>`](https://www.codecademy.com/resources/docs/html/tags/strong)
[`<q>`](https://www.codecademy.com/resources/docs/html/tags/q)
[`<picture>`](https://www.codecademy.com/resources/docs/html/tags/picture)
[h1-h6](https://www.codecademy.com/resources/docs/html/tags/h1-h6)
[`<form>`](https://www.codecademy.com/resources/docs/html/tags/form)
[`<em>`](https://www.codecademy.com/resources/docs/html/tags/em)
[`<cite>`](https://www.codecademy.com/resources/docs/html/tags/cite)
[`<button>`](https://www.codecademy.com/resources/docs/html/tags/button)
[`<br>`](https://www.codecademy.com/resources/docs/html/tags/br)
[`<blockquote>`](https://www.codecademy.com/resources/docs/html/tags/blockquote)
[`<audio>`](https://www.codecademy.com/resources/docs/html/tags/audio)
[`<a>`](https://www.codecademy.com/resources/docs/html/tags/a)
[`<details>`](https://www.codecademy.com/resources/docs/html/semantic-html/details)
[`<summary>`](https://www.codecademy.com/resources/docs/html/semantic-html/summary)

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by ethan-master-coding**
### Description

Added the output of examples for the picture of audio tags.

Closes #737

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by ethan-master-coding** [comment]
@Dusch4593 How should I do the audio tag since `gifs` don't support audio. Are we able to use the `<audio>` tag? If so can you get audio about CC?

**Comment by Dusch4593** [comment]
> @Dusch4593 How should I do the audio tag since `gifs` don't support audio. Are we able to use the `<audio>` tag? If so can you get audio about CC?

We are not able to use the `<audio>` tag, unfortunately. 😞

**Comment by ethan-master-coding** [comment]
@Dusch4593 Then what is the best way to add the audio tag example output then?

**Comment by Dusch4593** [comment]
Hey @ethan-master-coding let's hold off on this real quick and see my [comment](https://github.com/Codecademy/docs/pull/760#issuecomment-1140009940) at #760.

**Comment by ethan-master-coding** [comment]
@Dusch4593 I am not sure. I took it from the comment you send in #737. Mabey you can upload it so we can get it raw witch may be smaller.

**Comment by Dusch4593** [comment]
> @Dusch4593 Then what is the best way to add the audio tag example output then?

I double-checked and it seems that you are actually able to use `<audio>` tags 😄 Let's make sure to include that `controls` attribute, though.

File: content/html/concepts/tags/terms/video/video.md
Comment (Dusch4593) [inline content/html/concepts/tags/terms/video/video.md:57]:
```suggestion
<video controls>
  <source type="video/mp4" src="https://raw.githubusercontent.com/Codecademy/docs/main/media/html-video-tag-example.mp4" />
</video>
```

Comment (Dusch4593) [reply content/html/concepts/tags/terms/video/video.md:57]: Actually, let's stick with using the inner `<source>` tag but with `controls` added to outer `<video>` tag.

**Review by Dusch4593 — CHANGES_REQUESTED** [review]

**Comment by ethan-master-coding** [comment]
@Dusch4593 I made that change. And also I made the outline of the audio tag example but I need audio to use with it.

**Comment by Dusch4593** [comment]
> @Dusch4593 I am not sure. I took it from the comment you send in #737. Mabey you can upload it so we can get it raw witch may be smaller.

Actually, this may not be as pressing as I thought, according to my colleagues in Engineering.

**Comment by Dusch4593** [comment]
I'm going to look for something we can use this weekend. Hopefully we'll have some audio next week.

**Comment by ethan-master-coding** [comment]
@Dusch4593 If you can't find anything here's an idea. Have Sonny record some audio about the docs. Similar to the video.

**Comment by Dusch4593** [comment]
@ethan-master-coding I added an **.mp3** file with audio about Docs. Make sure to use the following format when adding to the `<audio>` tag: `https://raw.githubusercontent.com/Codecademy/docs/main/media/introducing_docs_audio.mp3`

**Comment by ethan-master-coding** [comment]
@Dusch4593 I added the audio. This is ready for review

**Review by Dusch4593 — APPROVED** [review]
@ethan-master-coding This LGTM! 👍🏻 Let's have one more review of the files before we merge.

File: content/html/concepts/tags/terms/audio/audio.md
Comment (KTom101) [inline content/html/concepts/tags/terms/audio/audio.md:51]: When viewing the file I noticed this displays as a header on the page. Is this what we are going for?

**Review by KTom101 — COMMENTED** [review]
@ethan-master-coding @Dusch4593 Looks good, just one comment from me.

File: content/html/concepts/tags/terms/audio/audio.md
Comment (ethan-master-coding) [reply content/html/concepts/tags/terms/audio/audio.md:51]: @KTom101 A guess that GH MD doesent support the audio tag so it goes back to the fallback audio.

Comment (Dusch4593) [reply content/html/concepts/tags/terms/audio/audio.md:51]: @ethan-master-coding I think we should just go with plain text and remove the `<h3>`s for both the `<video>` and `<audio>` example outputs.

Comment (Dusch4593) [reply content/html/concepts/tags/terms/audio/audio.md:51]:
```
<audio controls>
  <source src="https://raw.githubusercontent.com/Codecademy/docs/main/media/introducing_docs_audio.mp3" type="audio/mp3" />
  Fallback text if audio will not play
</audio>

Comment (ethan-master-coding) [reply content/html/concepts/tags/terms/audio/audio.md]: @Dusch4593 I made that change.

**Comment by Dusch4593** [comment]
@ethan-master-coding Merged! Let us know if the output comes out okay on your end. Thanks for your work on this 😄

**Pull request merged**
